terraform/gcp/modules/monitoring/fulcio/fulcio_alerts.tf

```diff
@@ -174,3 +174,75 @@ resource "google_monitoring_alert_policy" "ca_service_cert_quota" {
   notification_channels = local.notification_channels
   project               = var.project_id
 }
+
+### K8s Alerts
+
+resource "google_monitoring_alert_policy" "fulcio_k8s_pod_restart_failing_container" {
+  # In the absence of data, incident will auto-close in 7 days
+
+  alert_strategy {
+    auto_close = "604800s"
+  }
+
+  combiner = "OR"
+
+  conditions {
+    condition_threshold {
+      filter     = "metric.name=\"k8s_pod/restarting-failed-container\" resource.type=\"k8s_pod\""
```

Comment:
Do we need to include any fulcio specific details in this filter? Sorry if that's a dumb question I'm not super familiar with the logging alerts

Reply:
This filter was incorrect, the metric name portion should have been `fulcio/k8s_pod/restarting-failed-container`. It has been updated to reference this name.